tests/x11/remote_desktop/screensharing_available.pm

```diff
@@ -35,18 +34,23 @@ sub run {
     # It may happen that the screen sharing is not available
     assert_screen [qw(with_screensharing without_screensharing)];
     if (match_has_tag 'without_screensharing') {
-        record_info 'vino missing', 'After the installation the screen sharing is not available - vino is missing and we need to install it now.';
-        send_key 'ctrl-q';
-
-        # Install the vino package which is probably the case of missing screen sharing option
-        ensure_installed 'vino';
-
+        if (is_sle("15-sp4+")) {
+            record_info 'gnome-remote-desktop', 'After the installation the screen sharing is not available - gnome-remote-dekstop is missing and we need to install it now.';
```

Comment:
is there a bug for this? I think this should be a soft failure (same below)... I.E we should know what's the expectation and expected result after the installation?. Maybe a separate ticket to figure that out and update later (and actually use the screen sharing)... to keep things moving

Reply:
In SLE15 SP4, gnome-remote-desktop is available by default and so screen sharing, I added check similar to earlier condition. But it is gnome-remote-desktop instead of vino for SP4.